docs/file-reference.md

```diff
@@ -401,21 +401,61 @@ More info about Probe: https://kubernetes.io/docs/api-reference/v1.6/#probe-v1-c
 
 The name of the Ingress.
 
+## secrets
+
+```yaml
+secrets:
+- <secret>
+- <secret>
+```
+
+| **Type**                         | **Required** |
+|----------------------------------|--------------|
+| array of [secret](#secret) | no           |
+
+###secret
+
+```yaml
+name: string
+<Kubernetes Secret Definition>
+```
+
+The Kubernetes Secret resource is being reused here. More info: https://kubernetes.io/docs/api-reference/v1.6/#secret-v1-core
```

Comment:
Aas Kedge is able to "simplify" Kubernetes deployments, it's unclear what's happening here in the documentation. We may need to be more verbose that these are base64 values rather than giving a link to a whole other doc to read. People may be putting their actual credentials / secrets without base64'ing them :(